Dockerfile

```diff
@@ -70,6 +70,9 @@ RUN apk add --no-cache libstdc++ git curl openssh openssh-server bash rsync net-
 COPY --from=build $MZBENCH_API_DIR $MZBENCH_API_DIR
 COPY --from=build ${HOME_DIR}/.local ${HOME_DIR}/.local
 
+# SSH requires root to have a password. Otherwise it will prompt for an interactive password
+RUN echo "root:Docker!" | chpasswd && "PasswordAuthentication no" > /etc/ssh/sshd_config
```

Comment:
In order not to create an additional layer could you move these command to one-liner above, something like this?
  
```
&& chmod 0600 ${HOME_DIR}/.ssh/authorized_keys \ 
&& echo "root:Docker!" | chpasswd && "PasswordAuthentication no" > /etc/ssh/sshd_config \
&& pip install -r /tmp/requirements.txt \
```